src/resources/js/router.js

```diff
@@ -19,7 +20,14 @@ const router = new VueRouter({
       name: 'Test',
       icon: 'mdi-help-box',
       component: Hello,
-    }],
+    },
+    {
+      path: '/userEdit',
```

Comment:
You should change the route with the following rule `/<resource>/<id?>/<action>` here should be something like `/utilisateur/1/modifier` or `/mon-compte/modifier` and avoid to list the user id into the url as this form is to update the data about the current logged user

Reply:
ok, I changed the path name in [c2152d](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/c2152dde75f38fadda5f8193b5907887f93ca823) but in comment to avoid issue #128